File: experiment_jp_test/j2pyrawpy/if__2__CHECK_DIVISIBILITY_LARGE_NUMBER_999.py
#!/usr/bin/env python
""" generated source for module if__2__CHECK_DIVISIBILITY_LARGE_NUMBER_999 """
class X(object):
    """ generated source for class X """
    @classmethod
    def isDivisible999(cls, num):
        """ generated source for method isDivisible999 """
        n = len(num)
        gSum = 0
        if gSum > 1000:
            num = Integer.toString(gSum)
            n = len(num)
            gSum = 1 if isDivisible999(num) else 0
        return (gSum == 999)

File: experiment_jp_test/j2pyraw/if__2__CHECK_DIVISIBILITY_LARGE_NUMBER_999.java
class X { static boolean isDivisible999 ( String num ) { int n = num . length ( ) ; int gSum = 0 ; if ( gSum > 1000 ) { num = Integer . toString ( gSum ) ; n = num . length ( ) ; gSum = isDivisible999 ( num ) ? 1 : 0 ; } return ( gSum == 999 ) ; }
 }